src/integrationTest/sam.test.ts

```diff
@@ -276,7 +276,7 @@ describe.skip('SAM Integration Tests', async () => {
                     assertCodeLensReferencesSamTemplate(codeLens, samTemplatePath)
                 }).timeout(TIMEOUT)
 
-                it('invokes the Run Local CodeLens', async () => {
+                it.skip('invokes the Run Local CodeLens', async () => {
```

Comment:
Any reason for keeping this around? Ditto for next test.

Reply:
these tests will be restored. out of scope for this PR.

Reply:
Why restore them? This feature branch should kill these tests.

Reply:
running `sam` is the point of these tests. that's the part that will be restored.